src/operator/api/v1alpha3/webhooks.go

```diff
@@ -47,4 +146,152 @@ func (in *ClientIntents) SetupWebhookWithManager(mgr ctrl.Manager, validator web
 		Complete()
 }
 
-func (in *ClientIntents) Hub() {}
+func (in *ClientIntents) ConvertTo(dstRaw conversion.Hub) error {
+	dst := dstRaw.(*v2alpha1.ClientIntents)
+	dst.ObjectMeta = in.ObjectMeta
+	if dst.Spec == nil {
+		dst.Spec = &v2alpha1.IntentsSpec{}
+	}
+	dst.Spec.Workload.Name = in.Spec.Service.Name
+	dst.Spec.Workload.Kind = in.Spec.Service.Kind
+	dst.Spec.Targets = make([]v2alpha1.Target, len(in.Spec.Calls))
+	for i, call := range in.Spec.Calls {
+		if call.IsTargetInCluster() && call.Type != IntentTypeKafka {
```

Comment:
I went and looked what `IntentTypeHTTP` means, and saw that it means Type is in {"", IntentTypeHTTP, IntentTypeKafka}. 
And then this code exclude IntentTypeKafka. 
So really, in this case it would just be clearer to replace this line with: 
`if call.Type == "" || call.Type == IntentTypeHTTP`. 

And then, since 2 lines from how you split again to the case of IntentTypeHTTP, how about just:
```
if call.Type == "" {
    kubernetesTarget := v2alpha1.KubernetesTarget{Name: call.GetServerFullyQualifiedName(in.Namespace), Kind: call.GetTargetServerKind()}
    dst.Spec.Targets[i] = v2alpha1.Target{Kubernetes: &kubernetesTarget}
}
if call.Type == HTTP { 
  ....
}
```